### PR TITLE
Final Pull Request for Checkpoint 3

### PR DESCRIPTION
**Pull request by JonahBurke**

### CODE REVIEW

**Comment by JonahBurke** [comment]
Gonna try and resolve some merge conflicts first

**Comment by JonahBurke** [comment]
Merge conflicts resolved (hopefully). Reopened pull request

File: GroceryListService/Models/User.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace GroceryListService.Models
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; } // This might need to be changed for security reasons, but for now, we'll be okay
```
Comment (dnavarro3) [inline GroceryListService/Models/User.cs:7]: You will need some form of password encryption, one option might be to use PasswordHasher

**Review by dnavarro3 — COMMENTED** [review]
Jiakai doesnt seem to be listed as a project contributor. Everything else seems okay. Good luck with the final checkpoint. Be sure to make equitable contributions.

**Pull request merged**